### DISCUSSION TITLE
Globset README clarification

### DISCUSSION THREAD

**Discussion by hoijui** [BurntSushi/ripgrep#2087] [Q&A] [answered]
In the very end, the README states:

> * Doesn't provide a recursive directory iterator of matching file paths, although I believe this crate should grow one eventually.

When I read this, I though, this means that '**' is not supported, and I kept on looking for an other crate that does.
Now (an other day), I came back here, and was unsure if this is the same project I looked at last time,
so I read the README again, and saw this line:

```
builder.add(Glob::new("src/**/foo.rs")?);
```

Now I am confused.. what does the *recursive directory iterator* mean, which is not supported?

**Comment by hoijui** [comment, marked as answer]
... ahh I think I know. ;-)
one can only check if a path matches (like with a regex), but not get all matching paths.

**Reply by BurntSushi** [reply]
Correct.
